tests/funcalign/test_srm.py

```diff
@@ -117,6 +117,21 @@ def test_can_instantiate():
         s.fit(X)
     print("Test: different number of samples per subject")
 
+    # Check save/load functionality for fitted SRM
+    srm_fn = tmp_path
```

Comment:
You also need to add `tmp_path` as an argument to the function:
```
def test_can_instantiate(tmp_path):
```
Also, `tmp_path` is a directory, so you need a path to a file within, for example:
```
srm_fn = tmp_path / "srm.npz"
```
Finally, I usually read `fn` as `function`. How about `srm_path`?

Reply:
Okay, sounds good—I think I got all that fixed up